ui/tabs.js

```diff
@@ -664,24 +669,27 @@ return $.widget( "ui.tabs", {
 			"aria-selected": "false",
 			"aria-expanded": "false"
 		} );
+
 		// If we're switching tabs, remove the old tab from the tab order.
 		// If we're opening from collapsed state, remove the previous tab from the tab order.
 		// If we're collapsing, then keep the collapsing tab in the tab order.
 		if ( toShow.length && toHide.length ) {
 			eventData.oldTab.attr( "tabIndex", -1 );
 		} else if ( toShow.length ) {
-			this.tabs.filter( function() {
-				return $( this ).attr( "tabIndex" ) === 0;
-			} )
-				.attr( "tabIndex", -1 );
+			this.tabs
+				.filter( function() {
+					return $( this ).attr( "tabIndex" ) === 0;
+				} )
+					.attr( "tabIndex", -1 );
```

Comment:
I'd prefer the second. Here's the way I'd look at it: With jQuery chaining we have two types of patterns. 

1) Sequential code, which is represented here. Results may be filtered but only those filtered results are used later in the chain. Indenting doesn't do anything for us there, and is distracting.

2) Stacking/unstacking code, which occurs when there is some filtering operation and `.end()` is used further down the chain. Indenting is important there to mark the stack/unstack points.